src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -400,4 +400,51 @@ object messages {
            |  ((${nestedRepresentation}))""".stripMargin
     }
   }
+
+  case class RepeatedModifier(modifier: String)(implicit ctx:Context) extends Message(14) {
+    val kind = "Syntax"
+
+    val msg = hl"""repeated modifier $modifier"""
+
+    val code1 = hl"""private private val Origin = Point(0, 0)"""
+
+    val code2 = hl"""private final val Origin = Point(0, 0)"""
+
+    val explanation =
+      hl"""This happens when you accidentally specify the same modifier twice.
+           |
+           |Example:
+           |
+           |$code1
+           |
+           |instead of
+           |
+           |$code2
+           |
+           |""".stripMargin
```

Comment:
@liufengyun has been working on modifiers with positions, and I'm thinking maybe we could enhance this further to say make the following recommendation:

```
This happens when you accidentally specify the same modifier twice:

private private lazy val x = ...

instead of:

private lazy val x = ...
```

Where this would be the user's code without the duplicate modifier. What do you think @liufengyun - can this be done with the new modifier objects?